{"name": "test_app_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestSuite' object has no attribute 'app'", "trace": "self = <scripts.test05_app_login.TestSuite object at 0x0000000003C45550>, phone = '13812345678', code = '246810'\n\n    def test_app_login(self, phone='13812345678', code='246810'):\n        # 调用app登录业务方法\n>       self.app.page_app_login(phone, code)\nE       AttributeError: 'TestSuite' object has no attribute 'app'\n\nscripts\\test05_app_login.py:28: AttributeError"}, "start": 1596636050962, "stop": 1596636050962, "uuid": "a154d5c3-6ba4-4544-b145-16f41f5b9210", "historyId": "d32bbdd46c0c13171b1b742d3837ca68", "testCaseId": "bc361d1fdd7ceb4aac8ddb860da126fc", "fullName": "scripts.test05_app_login.TestSuite#test_app_login", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test05_app_login"}, {"name": "subSuite", "value": "TestSuite"}, {"name": "host", "value": "ETPUA4SBB4P3FBS"}, {"name": "thread", "value": "7448-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test05_app_login"}]}